tests/NovuTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Novu\SDK\Tests;
+
+use Novu\SDK\Novu;
+use Novu\SDK\Resources\Workflow;
+use PHPUnit\Framework\TestCase;
+
+class NovuTest extends Testcase
```

Comment:
The name of this class should probably be `WorkflowTest`. This class simply tests the Workflow feature, so the name should represent it.

Reply:
The way the SDK is designed, it contains all the services under one entity hence the `NovuTest`. Ofc, this contains all the workflow unit tests now but can now be later extended to cover all other services (triggers, notifications, etc)

If we want to keep it under a separate entity, we will have to discuss it a bit to avoid DRY principle

Reply:
All your service tests should not be in one class. You can extract them into traits or its individual classes like it's done in the SDK. If you don't we will have once class file littered with so many code methods that it will be so hard to read or  review @2002Bishwajeet